testsuite/single_node_performance.py

```diff
@@ -85,6 +85,13 @@ class RunGroupConfig:
 
     RunGroupConfig(expected_tps=50000, key=RunGroupKey("coin_transfer_connected_components", executor_type="sharded", sharding_traffic_flags="--connected-tx-grps 5000", transaction_type_override=""), included_in=Flow.REPRESENTATIVE),
     RunGroupConfig(expected_tps=50000, key=RunGroupKey("coin_transfer_hotspot", executor_type="sharded", sharding_traffic_flags="--hotspot-probability 0.8", transaction_type_override=""), included_in=Flow.REPRESENTATIVE),
+
+    RunGroupConfig(expected_tps=4000, key=RunGroupKey("resource-groups-global-resource"), included_in=Flow.LAND_BLOCKING, waived=True),
+    RunGroupConfig(expected_tps=8000, key=RunGroupKey("resource-groups-global-resource", module_working_set_size=20), included_in=Flow(0), waived=True),
+    RunGroupConfig(expected_tps=8000, key=RunGroupKey("resource-groups-individual-resource"), included_in=Flow.CONTINUOUS, waived=True),
+    RunGroupConfig(expected_tps=8000, key=RunGroupKey("resource-groups-individual-resource", module_working_set_size=20), included_in=Flow(0), waived=True),
+    RunGroupConfig(expected_tps=8000, key=RunGroupKey("resource-groups-multi-change"), included_in=Flow.LAND_BLOCKING, waived=True),
+    RunGroupConfig(expected_tps=8000, key=RunGroupKey("resource-groups-multi-change", module_working_set_size=20), included_in=Flow(0), waived=True),
```

Comment:
Does waived = True means the test passes even if the observed throughput is less than the expected throughput? Adding a comment on definition of waived flag would help.